src/angular/lightbox/lightbox-container.ts

```diff
@@ -64,6 +66,11 @@ export class SbbLightboxContainer extends _SbbDialogContainerBase {
     }
   }
 
+  @HostListener('window:resize', ['$event'])
```

Comment:
Ideally we could use the ScrollDispatcher from the CDK here, but this should work.

Reply:
I've used the `ViewportRuler`.